Website/pages/index.html

```diff
@@ -15,47 +15,43 @@
 
 </head>
 <body>
-  <h1>Text Area Above the Tables</h1>
-
-    <div class="table-container">
-
-      <div class="text-content" data-text="Bar Chart">
+    <div class="container-header">
+      <h1>Text Area Above the Tables</h1>
+      <a class="compare-btn" href="compare.html">Compare</a>
+    </div>
+    <div class="container">
+        <div class="chart-content">
+          <div class="text-content" data-text="Bar Chart">
             <h2> Commits per author</h2>
-      </div>
-
-        <!-- Plotly chart will be drawn inside this DIV -->
-      <div  id='myDiv'  >   </div>
-
-      <div class="description">
-        <p>This bar chart illustrates the number of commits made by individuals, with each bar representing a specific person and its height corresponding to their total commits. It provides a clear visual comparison of contribution levels among participants, highlighting variations in activity.</p>
-      </div>
-
-        <!-- Tables Side by Side -->
+          </div>
+          <div id='myDiv'></div>
+        </div>
         <div class="table-wrapper">
-            <!-- First Table -->
-            <table class="first-table">
+            <div class="description">
+              This bar chart illustrates the number of commits made by individuals, with each bar representing a specific person and its height corresponding to their total commits. It provides a clear visual comparison of contribution levels among participants, highlighting variations in activity.
+            </div>
+            <div class="table-block">
+              <table class="first-table">
```

Comment:
can we rename `first-table` and `second-table` below to something more sensible (maybe `highest-committers-table` and `lowest-committers-table`), and also make them IDs instead of classes?